src/applications/mhv/secure-messaging/components/AttachmentsList.jsx

```diff
@@ -64,16 +68,29 @@ const AttachmentsList = props => {
                       file.name
                     }`}
                   >
-                    <i className="fas fa-paperclip" aria-hidden="true" />
+                    <i
+                      className="fas fa-paperclip"
+                      alt="Attachment icon"
+                      aria-hidden="true"
+                    />
                     <span>{file.name} </span>(
                     {getSize(file.size || file.attachmentSize)})
                   </span>
-                  <va-button
-                    onClick={() => removeAttachment(file)}
-                    secondary
-                    text="REMOVE"
+                  <button
+                    type="button"
```

Comment:
is `type="button"` needed on `<button>` element?

Reply:
I think we still need this.. it is an explicit type and might upset the liner if we don't have it... :eyes: